tpm2_pytss/TSS2_Exception.py

```diff
@@ -1,2 +1,24 @@
+from .types import TPM2_RC
+
 class TSS2_Exception(RuntimeError):
-    pass
+    def __init__(self, rc):
+        super(RuntimeError, self).__init__(f"TSS2 Library call failed with: 0x{rc:X}")
```

Comment:
Wy don't we just bind it to rcdecode and get that string back and pass that up?

Reply:
Right now it's because I haven't gotten my head around cffi so I'm not ready to create bindings for libtss2-rc just yet, I can either mark this as WIP and fix it up when there is bindings for the RC decoder or if it's commited I'll fix fix it when there is bindings.

Reply:
I can do it, takes like 2 seconds in an ideal world :-p

Reply:
@whoo, looks good, you need to change
```python
super(RuntimeError, self).__init__(f"TSS2 Library call failed with: 0x{rc:X}")
```

with:
```python
super(TSS2_Exception, self).__init__(f"TSS2 Library call failed with: 0x{rc:X}")
```
to make LGTM happy.